### PR TITLE
113632387: Added placeholder for tornado app. "Hello World".

### PR DESCRIPTION
**Pull request by akellehe**

### CODE REVIEW

**Comment by akellehe** [comment]
@agirlnamedsophia please review

**Comment by akellehe** [comment]
https://www.pivotaltracker.com/story/show/113632387

**Comment by akellehe** [comment]
@agirlnamedsophia rebased

File: app.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+def make_app():
+    return tornado.web.Application([
+        (r"/", MainHandler),
+    ])
+
+if __name__ == '__main__':
+    app = make_app()
+    app.listen(8080)
+    tornado.ioloop.IOLoop.current().start()
```
Comment (agirlnamedsophia) [inline app.py]: @akellehe does running this file instantiate a simple HTTP server (tornado) on port 8080, and the ioloop is for handling events?

**Comment by agirlnamedsophia** [comment]
:+1:

File: app.py (hunk 1 above)
Comment (akellehe) [inline app.py]: Yeah! You got it.

**Pull request merged**